### ISSUE TITLE
Check if deaths <= cases <= tests

### ISSUE THREAD

**Issue by lazd** [covidatlas/coronadatascraper#185]
If data is provided the above check must be true, or the data is invalid

**Comment by jbencina** [comment]
I thought about this a bit more and it may not work for a single day. I think you'd have to look at this cumulatively since most places are reporting active cases while deaths and tests are cumulative.

**Comment by Wikunia** [comment]
Additionally check that deaths are not decreasing. At least then something needs to be double checked.

**Comment by JimBudde** [comment]
I don't think you can say the data is "invalid".  It is what it is at this point.  Reporting delays, etc are likely to cause some odd reporting of numbers.  I have simply been telling folks, expect to see odd numbers from time to time and hopefully it will clear up over a few days time, e.g. when cases were 'transferred' from cruise ships to countries.  my 2c.

**Comment by qgolsteyn** [comment]
If implemented, can be added as a step in the `steps` folder, most likely called `validate.js` or something similar.

**Comment by lgeorge-idm** [comment]
This may not work at all. 
* Deaths may be for individuals that were never tested, but were reported based on symptoms and/or post-mortem examination.
* # of tests can be # of samples or # of individuals. Best practice is to run at least 2 samples per individual.

**Comment by tautme** [comment]
Thank you all for thinking about this. I have been wanting to see number of test administered, because you can only have as many positives as you have tested.

**Comment by LukeBrantley** [comment]
Are tests marked 0 if there are no records of tests for that location on that day?

**Comment by tautme** [comment]
today I see 37154 NA's in the timeseries-tidy.csv. So there are both zeros and NA's. Now as to whether or not the original source is recording correctly the meaning of NA?

**Comment by piccolbo** [comment]
Test positive is one thing, but diagnostics criteria change, particularly when epidemic outpaces testing ability. At that point health authorities switch to other criteria such as any viral pneumonia,  x-ray abnormalities etc. Italy and China, afaik, have made the switch. Marching orders is Italy are "test if it has clinical implications". Typically if you are not severe, well oxygenated, just fever, cough and discomfort, they send you home under quarantine with palliative care. I think cases should be the best estimate there is, positive tests is a lower bound. If we insist on test-confirmed only, events my make our numbers irrelevant.
